docs/running-tests/android_webview.md

```diff
@@ -4,27 +4,42 @@ To run WPT on WebView on an Android device, some additional set-up is required.
 
 Currently, Android WebView support is experimental.
 
-* Please check [Chrome for Android](chrome_android) for the common
-  instructions for Android support first.
-
-* Install an up-to-date version of system webview shell:
-  * Go to
-  [chromium-browser-snapshots](https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Android/)
-  * Find the subdirectory with the highest number and click it.
-  * Download `chrome-android.zip` file and unzip it.
-  * Install `SystemWebViewShell.apk`.
-  * On emulator, system webview shell may already be installed by default. Then
-    you may need to remove the existing apk:
-     * Choose a userdebug build.
-     * Run an emulator with
-       [writable system partition from command line](https://chromium.googlesource.com/chromium/src/+/HEAD/docs/android_emulator.md/)
-
-* If you have an issue with ChromeDriver version, try removing
-  `_venv/bin/chromedriver` such that wpt runner can install a matching version
+## Prerequisites
+
+#### Please check [Chrome for Android](chrome_android.md) for the common instructions for Android support first.
+
+#### Ensure you have a userdebug or eng Android build installed on the device.
+
+#### Install an up-to-date version of system webview shell.
+1. Go to [chromium-browser-snapshots](https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Android/)
+2. Find the subdirectory with the highest number and click it, this number can be found
+   in the "Commit Position" column of row "LAST_CHANGE" (at bottom of page).
+3. Download `chrome-android.zip` file and unzip it.
+4. Install `SystemWebViewShell.apk`.
+5. On emulator, system webview shell may already be installed by default. Then you may need to remove the existing apk:
+   * Choose a userdebug build.
+   * Run an emulator with
+     [writable system partition from command line](https://chromium.googlesource.com/chromium/src/+/HEAD/docs/android_emulator.md/)
+
+#### If you have an issue with ChromeDriver version mismatch, try one of the following.
+  * Try removing `_venv/bin/chromedriver` such that wpt runner can install a matching version
   automatically. Failing that, please check your environment path and make
   sure that no other ChromeDriver is used.
+  * Download the [ChromeDriver binary](https://chromedriver.chromium.org/) matching your WebView's major version and specify it on the command line
```

Comment:
I think you can also use `wpt install chrome webdriver` or something, and we should make it work automatically for chrome_android from `wpt run`.

Reply:
Didn't know about this install command.  I just tested it, and no matter what channel I specify, it's always getting version 76, so looks like there's a bug with it, probably same root cause as this one: https://github.com/web-platform-tests/wpt/issues/18405